processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorProperties.java

```diff
@@ -179,6 +180,19 @@ public class ProcessorProperties extends AbstractDecatonProperties {
             PropertyDefinition.define("decaton.deferred.complete.timeout.ms", Long.class, -1L,
                                       v -> v instanceof Long);
 
+    /**
+     * Timeout for destroying processors.
+     * When {@link PartitionProcessor#close()} is called, all processors will be destroyed.
+     * At this time, Decaton waits synchronously for the running tasks to finish until this timeout.
+     * If it is not a problem not to wait for task completion, it is recommended to set this timeout,
```

Comment:
Actually, this feature should be used with caution because:
- When `PartitionProcessor` times out, Decaton will continue other clean-up tasks like destroying partition-scoped DecatonProcessor (https://github.com/line/decaton/blob/v6.2.0/processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/PartitionContext.java#L126), which could cause unexpected behavior in the middle of last `DecatonProcessor#process` which timed-out
- Also, since processor-thread may continue processing last task even after this timeout, the task may be completed at an unexpected timing after a while (e.g. after Decaton already released partition assignment for the task came from)

So this feature should be used only when users want to avoid waiting last task execution at the above risks.

So we should include these implications in the javadoc, and I think it's better not to use word "recommended" here.